ext/tidy/tidy.c

```diff
@@ -435,7 +443,7 @@ static const zend_function_entry tidy_functions[] = {
 };
 
 static const zend_function_entry tidy_funcs_doc[] = {
-	TIDY_METHOD_MAP(getOpt, tidy_getopt, arginfo_tidy_getopt)
+	TIDY_METHOD_MAP(getOpt, tidy_getopt, arginfo_tidy_getopt_method)
 	TIDY_METHOD_MAP(cleanRepair, tidy_clean_repair, NULL)
```

Comment:
For PHP 8, it would be good to replace these NULLs with explicit empty arginfo (can be one shared for all methods without args). No arginfo and empty arginfo are not the same (the former opts out of LSP checks).

Reply:
Will create PR for https://github.com/TysonAndre/php-src/pull/new/tidy-arginfo-expect-0-args against master after this PR is released